acceptance/tests/plan_ssh.rb

```diff
@@ -7,58 +8,95 @@
   ssh_nodes = select_hosts(roles: ['ssh'])
   skip_test('no applicable nodes to test on') if ssh_nodes.empty?
 
+  node = ssh_nodes[0].hostname.sub(/\.delivery\.puppetlabs\.net/, '')
```

Comment:
I think this can be ssh_nodes[0].hostname.split('.')[0] to be less location specific